src/core/SamplePlayHandle.cpp

```diff
@@ -139,7 +137,7 @@ bool SamplePlayHandle::isFromTrack( const Track * _track ) const
 
 f_cnt_t SamplePlayHandle::totalFrames() const
 {
-	return ( m_sampleBuffer->endFrame() - m_sampleBuffer->startFrame() ) * ( Engine::mixer()->processingSampleRate() / Engine::mixer()->baseSampleRate() );
+	return ( m_sampleBuffer->endFrame() - m_sampleBuffer->startFrame() ) * ( Engine::mixer()->processingSampleRate() / m_sampleBuffer->sampleRate ());
```

Comment:
Seems like a long-standing bug, but the denominator should be:
```cpp
(float)Engine::mixer()->processingSampleRate() / m_sampleBuffer->sampleRate()
```
This will fix empty export result issue.